bin/api/generate_api_key.py

```diff
@@ -0,0 +1,7 @@
+import os
+import binascii
+
+def generate_api_key():
+    return binascii.hexlify(os.urandom(16)).decode()
```

Comment:
Why do you want to generate API key with python not go? And what the API key is for? Is this that you talked in slack API key for authorization?

Reply:
The purpose of this script is written in README. I'll refacto to Go it